aiopg/connection.py

```diff
@@ -106,7 +106,8 @@ class Connection:
 
     def __init__(self, dsn, loop, timeout, waiter, echo, **kwargs):
         self._loop = loop
-        self._conn = psycopg2.connect(dsn, async=True, **kwargs)
```

Comment:
Here we can use `async_` keyword: http://initd.org/psycopg/docs/connection.html#connection.async_

Reply:
fixed